src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/NodesDnValidator.java

```diff
@@ -0,0 +1,16 @@
+package com.amazon.opendistroforelasticsearch.security.dlic.rest.validation;
+
+import org.elasticsearch.common.bytes.BytesReference;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.rest.RestRequest;
+
+public class NodesDnValidator extends AbstractConfigurationValidator {
+
+    public NodesDnValidator(final RestRequest request, boolean isSuperAdmin, final BytesReference ref, final Settings esSettings, Object... param) {
+        super(request, ref, esSettings, param);
```

Comment:
For PUT requests where API can enter free-form value for nodesdn you may want to validate the actual DN string. Eg: "CN=OpenDistro,OU=Server CA,O=OpenDistro,C=US" is acceptable but "xyz" is not.

Reply:
Good point, but I noticed there is no validation while loading values from ES YML. Let me look this up and accommodate the validation.

Reply:
I took another look and realized, we support both regex and wildcard, similar to how the YML value is consumed. So we won't be able to validate.